test/test_sparse_csr.py

```diff
@@ -1204,9 +1209,21 @@ def test_sparse_csr_consistency(self, device, dtype, op):
             if sample.input.ndim != 2:
                 continue
 
-            expected = op(sample.input)
+            expected = op(sample.input, *sample.args, **sample.kwargs)
+            sample.input = sample.input.to_sparse_csr()
+
+            # pow(Tensor, Tensor) is not supported yet for Sparse inputs
```

Comment:
So `pow` and `square` testing require a lot of special handling and they have separate tests in this PR.
I suggest filtering out `pow` and `square` from the `sparse_csr_unary_ufuncs` list so that this special handling is not needed here.